sources/packages/backend/workflow/src/deploy.ts

```diff
@@ -124,5 +124,7 @@ import {
     console.error("Error while executing the deployment.");
     console.error(error);
     throw error;
+  } finally {
+    zeebeClient.close();
```

Comment:
I believe that there is an `await` missing. Can you check, please?

Reply:
Yes it is a promise and added await.